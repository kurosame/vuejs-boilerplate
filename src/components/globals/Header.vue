<template>
  <div class="header">
    Header
  </div>
</template>
<style lang="scss" scoped>
.header {
  color: white;
  background-color: red;
}
</style>
<script lang="ts">
import Vue from 'vue'

export default Vue.extend({
  name: 'Header'
})
</script>
