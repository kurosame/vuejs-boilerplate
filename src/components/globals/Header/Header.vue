<template src="./header.html"></template>
<style module src="./header.scss"></style>
<script src="./header.js"></script>
