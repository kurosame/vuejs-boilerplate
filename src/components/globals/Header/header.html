<div class="header" :class="$style.header">
  Header
</div>
