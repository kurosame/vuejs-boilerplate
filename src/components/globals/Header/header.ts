import Vue from 'vue'

export default Vue.extend({
  name: 'Header'
})
