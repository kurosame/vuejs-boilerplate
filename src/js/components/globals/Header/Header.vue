<template src="./header.html"></template>
<style scoped lang="scss" src="./header.scss"></style>
<script src="./header.js"></script>
