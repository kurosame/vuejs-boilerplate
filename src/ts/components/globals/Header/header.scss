.header {
  color: white;
  background-color: red;
}
