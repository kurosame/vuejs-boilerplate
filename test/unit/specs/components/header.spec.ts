import Header from '@/components/Header.vue'
import { mount, Wrapper } from '@vue/test-utils'

let wrapper: Wrapper<Header>
beforeEach(() => {
  wrapper = mount(Header)
})

test('Match the snapshot', () => {
  expect(wrapper.html()).toMatchSnapshot()
})
