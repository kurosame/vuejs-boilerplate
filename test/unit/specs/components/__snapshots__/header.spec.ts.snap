// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Header.vue snapshot 1`] = `
<div class="header">
  Header
</div>
`;
